app-forensics/chkrootkit/chkrootkit-0.55-r1.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd toolchain-funcs
+
+GENTOO_PATCH="${PN}-0.55-gentoo.patch"
+
+DESCRIPTION="Tool to locally check for signs of a rootkit"
+HOMEPAGE="http://www.chkrootkit.org/"
+SRC_URI="ftp://ftp.pangeia.com.br/pub/seg/pac/${P}.tar.gz"
+SRC_URI+=" https://dev.gentoo.org/~sam/distfiles/${CATEGORY}/${PN}/${GENTOO_PATCH}.bz2"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+IUSE="+cron systemd"
+
+DEPEND="systemd? ( sys-apps/systemd )"
```

Comment:
Don't add a systemd USE flag or dependency just for a [small file](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301) (although you're unconditionally installing the unit file, so it's just being used for pkg_postinst, actually).